spec.html

```diff
@@ -1679,7 +1679,7 @@ <h1>Number::leftShift ( _x_, _y_ )</h1>
           <emu-alg>
             1. Let _lnum_ be ! ToInt32(_x_).
             1. Let _rnum_ be ! ToUint32(_y_).
-            1. Let _shiftCount_ be the result of masking out all but the least significant 5 bits of _rnum_, that is, compute _rnum_ &amp; 0x1F.
+            1. Let _shiftCount_ be _rnum_ modulo 32.
```

Comment:
`modulo` is an abstract operation
```suggestion
            1. Let _shiftCount_ be modulo(_rnum_, 32).
```

Reply:
Is that incoming? There are currently 35 uses of modulo, all of which look like this ([example](https://tc39.es/ecma262/#sec-bigintbitwiseop)).

Reply:
oh oops, you're totally right; disregard this.